src/container/srv_target.c

```diff
@@ -752,6 +752,9 @@ ds_cont_child_cache_destroy(struct daos_lru_cache *cache)
 static void
 cont_child_put(struct daos_lru_cache *cache, struct ds_cont_child *cont)
 {
+	if (unlikely(cont->sc_dtx_reindex && daos_lru_left_last_user(&cont->sc_list)))
+		stop_dtx_reindex_ult(cont, false);
```

Comment:
DTX reindex can be triggered when container open, also can be triggered by DTX resync. If we stop DTX reindex when cont_child_stop(), it may affect in-process DTX resync.